data/common.yaml

```diff
@@ -1,9 +1,43 @@
 ---
-profile_session_log::enable_session_log: true
 
-profile_session_log::rsyslog::forward_protocol: "relp-tls"
+profile_session_log::enable: false
+
+profile_session_log::rsyslog::forward_protocol: "tcp-tls"
 profile_session_log::rsyslog::remote_syslog_server: "tlog.security.ncsa.illinois.edu"
-profile_session_log::rsyslog::remote_syslog_server_port: 6516
+profile_session_log::rsyslog::remote_syslog_server_port: 6514
 
+profile_session_log::sssd::exclude_users:
+  - "qualys"
 profile_session_log::sssd::groups: []
 profile_session_log::sssd::users: []
+
+profile_session_log::tlog::conf: |
```

Comment:
Could this be a template with parameters?
How much actually might change on a different system?

Reply:
See other comment, turned this into a template